test-requirements-py27.txt

```diff
@@ -0,0 +1,35 @@
+# NOTE: codecov only supports coverage==4.5.2
+coverage==4.5.2
+pep8==1.7.1
+st2flake8==0.1.0
+astroid==1.6.5
+pylint==1.9.4
+pylint-plugin-utils>=0.4
+bandit==1.5.1
+ipython<6.0.0
+mock==3.0.5
+nose>=1.3.7
+tabulate
+unittest2
+sphinx==1.7.9
+sphinx-autobuild
+# nosetests enhancements
+rednose
+nose-timer==0.7.5
+# splitting tests run on a separate CI machines
+nose-parallel==0.3.1
+# Required by st2client tests
+pyyaml==5.1.2
+RandomWords
+gunicorn==19.9.0
+psutil==5.6.6
+webtest==2.0.25
+rstcheck>=3.3.1,<3.4
+tox==3.14.1
+pyrabbit
+# Since StackStorm v2.8.0 we now use cryptography instead of keyczar, but we still have some tests
```

Comment:
A query - if keyczar tests only run on python2.7, then as these are split requirement files, should it get removed from the test-requirements.txt?